src/themes/base.js

```diff
@@ -17,12 +17,19 @@ export default {
     textAlign: 'center',
     '@media screen and (min-width:64em)': {
       fontSize: '32px',
-    }
+    },
+    'li > ul, li > ol': {
```

Comment:
This selector will only target direct children. Is that intentional?

Reply:
I only targeted what was necessary to fix the generated nested lists as to try to reduce any unexpected behaviour. I don't want to mess up any custom html that people may have used.

Reply:
This should work as intended. Any nested list should be the direct child of an li, right?

Reply:
Correct.